control-plane-operator/controllers/hostedcontrolplane/cloud/powervs/powervs.go

```diff
@@ -174,7 +174,7 @@ func ReconcileCCMDeployment(deployment *appsv1.Deployment, hcp *hyperv1.HostedCo
 						Name: hcp.Status.KubeConfig.Name,
```

Comment:
can you change the name here and the name in the VolumeMounts section as well?

Reply:
Renamed in other places.